### PR TITLE
Post log

### PR DESCRIPTION
**Pull request by tomasbedrich**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -94,6 +96,24 @@ Using lazy loading:
 The difference is, that ``Cache`` object is created immediately and the
 page is loaded when needed (accessing the name).
 
+Post a new log to a cache:
```
Comment (tomasbedrich) [inline README.rst]: Could you please make this a heading? Using: `~~~`

File: pycaching/cache.py [hunk 2]
```diff
@@ -412,6 +424,7 @@ def load(self):
         else:
             trackable_page_url = None
 
+        log_page_url = root.findAll("div", "CacheDetailNavigation NoPrint")[0].find("a", {'id' : 'ctl00_ContentBody_GeoNav_logButton'})["href"]
```
Comment (tomasbedrich) [inline pycaching/cache.py]: Could you select this element by using its ID? The ID should be unique in the whole page, so the selector could be shortened to: `root.find(id="ctl00_ContentBody_GeoNav_logButton")["href"]`

Comment (forslund) [inline pycaching/cache.py]: Thanks, this is so much better!

**Comment by forslund** [comment]
When we feel the this is ready for merge maybe I should do some rebasing and try to make a cleaner pull request?

**Comment by tomasbedrich** [comment]
I think it is ready for merge now. Do you want me to do it, or do you plan to add more features?

**Comment by forslund** [comment]
I don't plan to add more features right now. When I get hold of a travelbug I'll see if I can add logging of those but I think it's better to do that as a separate pull-request. Do you want me to clean up the pull request before merge or is it OK with all the one-line fix-commits in here?

**Comment by tomasbedrich** [comment]
I think it is OK, thank you!

**Pull request merged**